### PR TITLE
Instead of http status 203 append header

### PR DESCRIPTION
**Pull request by mjarco**

### CODE REVIEW

File: storages/merger/bucket_meta_data.go [hunk 1]
```diff
@@ -17,17 +22,22 @@ func MergePartially(firstResponse backend.Response, successes []backend.Response
 	if err != nil {
 		log.Printf("Could not read first response body reqID %s, reason %s", firstResponse.ReqID(), err)
 	}
-	firstResponse.DiscardBody()
-	statusCode := http.StatusOK
+	err = firstResponse.DiscardBody()
+	if err != nil {
+		log.Printf("Cannot close first response body reqId %s, reason: %s", firstResponse.ReqID(), err)
```
Comment (wookie41) [inline storages/merger/bucket_meta_data.go]: Doest `DiscardBody` closes the body?

File: storages/merger/bucket_meta_data.go [hunk 2]
```diff
@@ -17,17 +22,22 @@ func MergePartially(firstResponse backend.Response, successes []backend.Response
 	if err != nil {
 		log.Printf("Could not read first response body reqID %s, reason %s", firstResponse.ReqID(), err)
```
Comment (wookie41) [inline storages/merger/bucket_meta_data.go]: Shouldn't there be a `return`?

File: storages/merger/bucket_meta_data.go (hunk 1 above)
Comment (wookie41) [inline storages/merger/bucket_meta_data.go]: `Cannot close` -> `Cannot discard`
And does `DiscardBody` closes the `body` in case it can't be discarded?

Comment (mjarco) [reply storages/merger/bucket_meta_data.go]: Tries to, if ends up with error returns it

File: storages/merger/bucket_meta_data.go (hunk 2 above)
Comment (mjarco) [reply storages/merger/bucket_meta_data.go]: No, we want to return any response we have

**Pull request merged**